### ISSUE TITLE
Wrong url to download `libstdc++-devel` CentOS8 agent installation from sources

### ISSUE THREAD

**Issue by mcarmona99** [wazuh/wazuh-documentation#4984]
This issue was opened after reviewing the **Detecting and removing malware using VirusTotal integration** proof of concept guide in the [active response manual tests issue (Wazuh 4.3.0)](https://github.com/wazuh/wazuh/issues/12870). The full review of the proof 
 of concept can be found at https://github.com/wazuh/wazuh/issues/12870#issuecomment-1082002324.

In that POC, the Wazuh environment used is composed of a Wazuh manager, with a CentOS8 agent connected.

I have used the Wazuh 4.3.0 OVA for the manager.
For the Wazuh agent, I have followed the installation from sources guide (**CentOS8**, `yum`): `Documentation > Installation alternatives > Installation from sources > Installing Wazuh agent from sources`

In the agent installation, I have detected an error. In step 1, we install development tools and compilers:

```
# yum install make gcc gcc-c++ python3 python3-policycoreutils automake autoconf libtool openssl-devel cmake
# rpm -i $(rpm -q libstdc++-devel --queryformat "http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libstdc++-static-%{VERSION}-%{RELEASE}.%{arch}.rpm\n")
```

The first command worked, but the second didn't. In my case (CentOS8), changing from `mirror` to `vault` made the error disappear. This is due to the CentOS8 end of life. We need to use vault as it is the mirror provided for archived versions.

In other words, http://mirror.centos.org/centos/8/PowerTools is unreachable, whereas http://vault.centos.org/centos/8/PowerTools works.

This issue aims to change this command and use the `vault` mirror. Also, consider talking about the necessary changes in the `yum` repos used.

Note that this issue was discovered in the Wazuh v4.3.0 testing process, but it is also present in the 4.2 docu and other versions' (as I said, due to CentOS8 EOL). Ref: https://documentation.wazuh.com/current/installation-guide/more-installation-alternatives/wazuh-from-sources/wazuh-agent/index.html

https://www.centos.org/centos-linux-eol/:

> CentOS Linux 8 will reach End Of Life (EOL) on December 31st, 2021. Here’s what that means.
